DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -590,9 +590,13 @@ function Test-WebsiteBindings
     # Assume bindings do not need updating
     $BindingNeedsUpdating = $false
 
-    Write-Verbose "Getting Web Binding: $name"
+    <#
+        Currently there is a problem in the LCM where Get-WebBinding short circuts the verbose stream
+        Write-Log can be changed to add the -File switch to log to a directory for troubleshooting.
+        However it's pretty noisy so it's being left in but commented out.
+    #>
+
```

Comment:
@TravisEz13 could explain it better then I could. It's something to do with how Get-WebBinding shells out its call in the IIS module.

As for the extra logging I guess I could leave it, it felt uncessary but useful due to the bug with the verbose stream. I'll uncomment them and add the file switch.